src/Nest/Ingest/Processor/ElasticClient-IngestProcessorGrok.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	using System.Threading;
+	using NodesHotThreadConverter = Func<IApiCallDetails, Stream, NodesHotThreadsResponse>;
+
+	public partial interface IElasticClient
+	{
+		/// <summary>
+		/// Retrieving which patterns which the grok processor is packaged with, useful as different versions are bundled with different processors.
```

Comment:
_pedant alert!_: _Retrieving which patterns the grok processor_

Reply:
Aunglish.